### ISSUE TITLE
AirBnb Price Recommendations: Based on Sq Feet? Same for most Homes

### ISSUE THREAD

**Issue by MappingKat** [MappingKat/issues#12]
Most homes seem to suggest to rent $300 per night

**Comment by imcmahon** [comment]
Yeah, it's just a static recommendation currently and defaults to $125 for single and $300 for full home. The guy Ernie was meeting with last week was from AirDNA. We're working with them to get accurate rental estimates hopefully near future.
